### PR TITLE
version update for cut release

### PR DESCRIPTION
**Pull request by ynzhang0509**

### CODE REVIEW

**Comment by khandpur** [comment]
Please ensure you've updated (added or removed) new / old actions, assertions, matchers from the cheatsheet.

Also there's a test failure:
https://travis-ci.org/google/EarlGrey/jobs/351588720

Can you file an issue for it and/or investigate if it's easy fix?

**Comment by bootstraponline** [comment]
The new cheatsheet PNG size is kind of small.

**Comment by ynzhang0509** [comment]
Reran https://travis-ci.org/google/EarlGrey/jobs/351588720 and all test passed. Actually it is first time I saw that test fail. Will create a ticket internally and do some investigation to see if we can find the source of flakiness.

**Comment by tirodkar** [comment]
LG. Can you please fix the image problem please?

**Comment by khandpur** [comment]
@ynzhang0509 let us know when it's ready for review.

**Comment by ynzhang0509** [comment]
Looks like Chrome screenshot has some problem and generates smaller image than before. Changing window size in script does not help. Will investigate more about it.

**Comment by bootstraponline** [comment]
> Looks like Chrome screenshot has some problem and generates smaller image than before. Changing window size in script does not help. Will investigate more about it.

What Chrome version are you using? I tested on macOS 10.13.3 and Chrome 65.0.3325.162. No problems.

**Comment by ynzhang0509** [comment]
> What Chrome version are you using? I tested on macOS 10.13.3 and Chrome 65.0.3325.162. No problems.
The same Chrome version as you. But when I run render.sh I encountered some permission errors like:
[0319/134346.237396:ERROR:xattr.cc(64)] setxattr org.chromium.crashpad.database.initialized on file /var/folders/l2/gz5scmws3hn2q1y77lghvl50007xs6/T/: Operation not permitted (1)
[0319/134346.238083:INFO:crashpad_client_mac.cc(292)] restarting handler in 0.985s
[0319/134346.238136:ERROR:xattr.cc(64)] setxattr org.chromium.crashpad.database.initialized on file /var/folders/l2/gz5scmws3hn2q1y77lghvl50007xs6/T/: Operation not permitted (1)
[0319/134346.274133:ERROR:gpu_process_transport_factory.cc(1019)] Lost UI shared context.
[0319/134347.245339:ERROR:xattr.cc(64)] setxattr org.chromium.crashpad.database.initialized on file /var/folders/l2/gz5scmws3hn2q1y77lghvl50007xs6/T/: Operation not permitted (1)
[0319/134347.245911:INFO:crashpad_client_mac.cc(292)] restarting handler in 0.979s
[0319/134347.459573:INFO:headless_shell.cc(565)] Written to file /Users/ynzhang/Workspace/google_sync/EarlGrey/docs/cheatsheet/cheatsheet.png.
Not sure if ti is related though.

**Comment by bootstraponline** [comment]
>  ~/code/EarlGrey/Scripts/cheatsheet $ ./render.sh
> '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome' --headless --hide-scrollbars --disable-gpu --screenshot=/Users/user/code/EarlGrey/docs/cheatsheet/cheatsheet.png --window-size=1024,2550 file:///Users/user/code/EarlGrey/docs/cheatsheet/cheatsheet.html
>
> [0319/185939.898646:ERROR:gpu_process_transport_factory.cc(1019)] Lost UI shared context.
>
> [0319/185941.241965:INFO:headless_shell.cc(565)] Written to file > /Users/user/code/EarlGrey/docs/cheatsheet/cheatsheet.png.


Something is broken with your chrome. I'm not sure what. Works for me.

**Comment by bootstraponline** [comment]
looks like a known bug with mac SIP
https://stackoverflow.com/questions/49103799/running-chrome-in-headless-mode

> I suspect it's due to the SIP(System Integrity Protection) feature in MacOS, with that restrict, you can't modify the files under /var folder even you have the root privilege.

File: Scripts/cheatsheet/render.sh [hunk 1]
```diff
@@ -23,4 +23,4 @@ CHEATSHEET_PNG="$CHEATSHEET_DIR/cheatsheet.png"
 
 set -x
 
-"$CHROME" --headless --hide-scrollbars --disable-gpu --screenshot="$CHEATSHEET_PNG" --window-size=1024,2550 "file://$CHEATSHEET_HTML"
```
Comment (bootstraponline) [inline Scripts/cheatsheet/render.sh]: This change should be reverted.

**Pull request merged**